.circleci/tedana_outputs_verbose.txt

```diff
@@ -1,45 +1,60 @@
-dn_ts_OC.nii
-dn_ts_e1.nii
-dn_ts_e2.nii
-dn_ts_e3.nii
-dn_ts_e4.nii
-dn_ts_e5.nii
-feats_OC2.nii
-hik_ts_OC.nii
-hik_ts_e1.nii
-hik_ts_e2.nii
-hik_ts_e3.nii
-hik_ts_e4.nii
-hik_ts_e5.nii
-lowk_ts_OC.nii
-lowk_ts_e1.nii
-lowk_ts_e2.nii
-lowk_ts_e3.nii
-lowk_ts_e4.nii
-lowk_ts_e5.nii
+p06.SBJ01_S09_Task11_desc-TEDICAAcceptedZ_components.nii.gz
+p06.SBJ01_S09_Task11_desc-TEDICAAccepted_components.nii.gz
+p06.SBJ01_S09_Task11_desc-TEDICA_components.nii.gz
+p06.SBJ01_S09_Task11_desc-TEDICA_comptable.tsv
+p06.SBJ01_S09_Task11_desc-TEDICA_mixing.tsv
+p06.SBJ01_S09_Task11_desc-TEDPCAState_variables.pkl
+p06.SBJ01_S09_Task11_desc-TEDPCA_components.nii.gz
+p06.SBJ01_S09_Task11_desc-TEDPCA_comptable.tsv
+p06.SBJ01_S09_Task11_desc-TEDPCA_mixing.tsv
+p06.SBJ01_S09_Task11_desc-adaptiveGoodSignal_mask.nii.gz
+p06.SBJ01_S09_Task11_desc-ascendingEstimates_s0.nii.gz
+p06.SBJ01_S09_Task11_desc-ascendingEstimates_t2s.nii.gz
+p06.SBJ01_S09_Task11_desc-full_s0.nii.gz
+p06.SBJ01_S09_Task11_desc-full_t2s.nii.gz
+p06.SBJ01_S09_Task11_desc-initialTEDICA_mixing.tsv
+p06.SBJ01_S09_Task11_desc-limited_s0.nii.gz
+p06.SBJ01_S09_Task11_desc-limited_t2s.nii.gz
+p06.SBJ01_S09_Task11_desc-optcomAccepted_bold.nii.gz
+p06.SBJ01_S09_Task11_desc-optcomDenoised_bold.nii.gz
+p06.SBJ01_S09_Task11_desc-optcomRejected_bold.nii.gz
+p06.SBJ01_S09_Task11_desc-optcom_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-1_desc-accepted_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-1_desc-denoised_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-1_desc-rejected_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-2_desc-accepted_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-2_desc-denoised_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-2_desc-rejected_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-3_desc-accepted_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-3_desc-denoised_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-3_desc-rejected_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-4_desc-accepted_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-4_desc-denoised_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-4_desc-rejected_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-5_desc-accepted_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-5_desc-denoised_bold.nii.gz
+p06.SBJ01_S09_Task11_echo-5_desc-rejected_bold.nii.gz
+
+p06.SBJ01_S09_Task11_desc-TEDICAPredictedR2Values_something.nii.gz
+p06.SBJ01_S09_Task11_desc-TEDICAPredictedS0Values_something.nii.gz
+p06.SBJ01_S09_Task11_echo-1_desc-TEDICA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-2_desc-TEDICA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-3_desc-TEDICA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-4_desc-TEDICA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-5_desc-TEDICA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-1_desc-TEDPCA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-2_desc-TEDPCA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-3_desc-TEDPCA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-4_desc-TEDPCA_components.nii.gz
+p06.SBJ01_S09_Task11_echo-5_desc-TEDPCA_components.nii.gz
+
+
 meica_R2_pred.nii
 meica_S0_pred.nii
 meica_betas_catd.nii
 meica_metric_weights.nii
```

Comment:
I've got no idea what to call the metric_weights files either. They are used for weighted averaging across the brain, so in a sense they're like a nonbinary mask of some kind.

Reply:
I went with `_desc-metricAveragingTEDICA_weights.nii.gz`